content/release-10-15-0/streaming-analytics-10-15-0-bundle/10_15_0.md

```diff
@@ -12,22 +12,25 @@ in the Apama documentation.
 
 ### Improvements for multi-tenancy
 
-Apama custom microservices can now be multi-tenant.
+Apama applications can now work in a multi-tenant manner when used in a custom microservice.
 See also the information on running and testing the monitor file in [Developing apps with Software AG Designer](https://cumulocity.com/guides/apama/analytics-introduction/#sag-designer)
 and the required settings in the microservice manifest in [Deploying Apama applications as microservices](https://cumulocity.com/guides/apama/analytics-introduction/#deploying-as-microservice),
 both in the *Streaming Analytics guide*.
 
 ### Improvements for requests with concurrent connections
 
-In order to provide better performance in requests to the Cumulocity IoT platform, you can now configure the transport to use multiple client connections to perform requests concurrently. 
-You do this using the `client.numClients` tenant option in the `streaminganalytics` category.
+In order to provide better performance for requests to the Cumulocity IoT platform, Streaming Analytics now uses multiple client connections to perform requests concurrently. 
+You can adjust the default number of connections using the `client.numClients` tenant option in the `streaminganalytics` category.
 See also [Optimizing requests with concurrent connections](https://cumulocity.com/guides/apama/advanced/#optimize-requests) in the *Streaming Analytics guide*.
 
+The new concurrent connection capability provides improved performance for all smart rules, Analytics Builder models and EPL apps that were previously restricted in performance 
```

Comment:
Hmm, per-tenant and multi-tenant smartrules microservice don't support the tenant options and uses single client. Should we call it out ?